pkg/ingester/transfer.go

```diff
@@ -83,16 +83,17 @@ func (i *Ingester) TransferChunks(stream client.Ingester_TransferChunksServer) e
 			level.Info(util.Logger).Log("msg", "processing TransferChunks request", "from_ingester", fromIngesterID)
 		}
 		userCtx := user.InjectOrgID(stream.Context(), wireSeries.UserId)
+
 		descs, err := fromWireChunks(wireSeries.Chunks)
 		if err != nil {
 			return err
 		}
 
-		state, fp, series, err := userStates.getOrCreateSeries(userCtx, wireSeries.Labels)
+		var record Record // for the WAL, not used.
```

Comment:
If it's not used, why not just pass `nil`?

Reply:
Yup. Forgot to remove it in the cleanup.